var/spack/repos/builtin/packages/libdc1394/package.py

```diff
@@ -10,17 +10,26 @@ class Libdc1394(AutotoolsPackage):
     """Library providing an API for IEEE 1394 cameras."""
 
     homepage = "https://damien.douxchamps.net/ieee1394/libdc1394/"
-    url      = "https://downloads.sourceforge.net/project/libdc1394/libdc1394-2/2.2.6/libdc1394-2.2.6.tar.gz"
+    url = "https://downloads.sourceforge.net/project/libdc1394/libdc1394-2/2.2.6/libdc1394-2.2.6.tar.gz"
 
-    maintainers = ['traversaro']
+    maintainers = ["traversaro"]
 
-    version('2.2.6', sha256='2b905fc9aa4eec6bdcf6a2ae5f5ba021232739f5be047dec8fe8dd6049c10fed')
+    version(
+        "2.2.6",
+        sha256="2b905fc9aa4eec6bdcf6a2ae5f5ba021232739f5be047dec8fe8dd6049c10fed",
+    )
 
-    depends_on('libusb')
+    depends_on("pkgconfig", type="build")
+    depends_on("libusb")
+    depends_on("sdl2")
```

Comment:
If I am not wrong, `sdl`  is just a dependency for the examples, that are disabled in this recipe, no?

Reply:
@traversaro Looking at the library links, I believe you are correct. I will remove that dependency.